packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -102,13 +102,13 @@ const DailyVideoApiAdapter = (): VideoApiAdapter => {
     });
   }
 
-  const translateEvent = (event: CalendarEvent) => {
+  const translateEvent = async (event: CalendarEvent) => {
     // Documentation at: https://docs.daily.co/reference#list-rooms
     // added a 1 hour buffer for room expiration
     const exp = Math.round(new Date(event.endTime).getTime() / 1000) + 60 * 60;
-    const scalePlan = process.env.DAILY_SCALE_PLAN;
+    const { scale_plan } = await getDailyAppKeys();
```

Comment:
```suggestion
    const { scale_plan: scalePlan } = await getDailyAppKeys();
```

Reply:
You can do renaming and then leave everything as it was before.

Reply:
Also for this, whats the plan for self hosted? What's the way to turn scale_plan to true? from an env var? I had to modify directly the DB.

Reply:
@alannnc the env var which was used previously was always returning undefined.
did you run these commands after changing scale_plan to true 
cd packages/prisma
yarn seed-app-store ?

i didn't had to modify the db the api_key and scale_plan gets updated  after running this command.

Reply:
You are probably right, I didn't validate that command. just yarn dx